source/ftrack_api/data.py

```diff
@@ -81,7 +81,7 @@ def flush(self):
 
     def seek(self, offset, whence=os.SEEK_SET):
         """Move internal pointer by *offset*."""
-        self.wrapped_file.seek(offset, whence)
+        return self.wrapped_file.seek(offset, whence)
```

Comment:
consider updating parent class doc string to indicate that position is returned.